howto/components-scopes/README.md

```diff
@@ -17,7 +17,7 @@ apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
   name: statestore
-  namespace: production
```

Comment:
This one should not be changed. The example is using production namespace

Reply:
done